### PR TITLE
Y-axis range getting changed when same data passed during reflow.

### LINKED ISSUE #65: Y-axis range getting changed during reflow
# Bug Report

## Description

Y-axis range is getting updated while reflow for a graph that has autocalibration set to true. This is happening because we are updating [axisRange](https://github.com/cerner/terra-graphs/blob/main/packages/carbon-graphs/src/js/controls/Graph/Graph.js#L461) for each dataset with reflow. 

## Steps to Reproduce

1. Create a graph with 2 data series and set the autocalibration to true
2. Update the y-axis value via the reflow method
3. Y-axis range will get changed each time the reflow method is getting called, even though the data remains the same.

## Additional Context / Screenshots

Before Panning:
Screenshot 2021-01-20 at 10 14 35 PM
After Panning, the y-axis limit gets updated each time we click on the panning button:
Screenshot 2021-01-20 at 10 17 21 PM
Screenshot 2021-01-20 at 10 17 46 PM
Screenshot 2021-01-20 at 10 18 00 PM

## Expected Behavior

y-axis range should not get updated when the data is same

### PR DESCRIPTION
**Pull request by AshishMotanamGurunadham**
### Summary

Y-axis range is getting updated during reflow on each click when reflow graph data contains ‘0’ as it’s one of the y values. Made small changes to range logic to fix this issue.

Closes #65 

### Deployment Link

https://terra-graphs-deployed-pr-#.herokuapp.com/

### Testing

- Changes has been verified by testing range modified parameter after reflow is done.
### Additional Details
**Screenshots:** 

**Before:**
- Below images are captured after clicking panning button.
 
image

image

image

**After:**

image

image

image

Thank you for contributing to Terra.
@cerner/terra
@cerner/carbon

### CODE REVIEW

File: packages/carbon-graphs/CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 * Changed
   * Fixed linter issues.
+  * Y-axis range getting changed when same data passed during reflow.
```
Comment (sdadn) [inline packages/carbon-graphs/CHANGELOG.md:7]: nit: this should be a statement of the code change, not the issue. 
e.g: `fixed Y-axis reflow bug`

File: packages/carbon-graphs/tests/unit/controls/Line/LinePanning-spec.js [hunk 2]
```diff
@@ -424,5 +424,26 @@ describe('Line - Panning', () => {
         expect(legendItem.getAttribute('aria-current')).toBe('true');
       });
     });
+    describe('when same data is passed on multiple clicks of panning', () => {
```
Comment (sdadn) [inline packages/carbon-graphs/tests/unit/controls/Line/LinePanning-spec.js:427]: nit: when *the* same data

File: packages/carbon-graphs/tests/unit/controls/Bubble/SingleDataset/BubbleSingleDatasetPanning-spec.js [hunk 3]
```diff
@@ -351,5 +351,26 @@ describe('Bubble Single Dataset - Panning', () => {
         expect(legendItem.getAttribute('aria-current')).toBe('true');
       });
     });
+    describe('when same data is passed on multiple clicks of panning', () => {
```
Comment (sdadn) [inline packages/carbon-graphs/tests/unit/controls/Bubble/SingleDataset/BubbleSingleDatasetPanning-spec.js:354]: nit: when *the* same data

File: packages/carbon-graphs/tests/unit/controls/Bubble/MultipleDataset/BubbleMultipleDatasetPanning-spec.js [hunk 4]
```diff
@@ -350,5 +350,26 @@ describe('Bubble Multiple Dataset- Panning', () => {
         expect(legendItem.getAttribute('aria-current')).toBe('true');
       });
     });
+    describe('when same data is passed on multiple clicks of panning', () => {
```
Comment (sdadn) [inline packages/carbon-graphs/tests/unit/controls/Bubble/MultipleDataset/BubbleMultipleDatasetPanning-spec.js:353]: nit: when *the* same data

**Review by mraplumb — APPROVED** [review]

File: packages/carbon-graphs/CHANGELOG.md (hunk 1 above)
Comment (AshishMotanamGurunadham) [reply packages/carbon-graphs/CHANGELOG.md]: Fixed [here](https://github.com/cerner/terra-graphs/pull/66/commits/1f556c88eae25ac272b14101b4dd705321f965ad#diff-7b73f4b73c95577a16e8a58d2e1377b2204811d315c020884e80b9f22128c42aR7).

File: packages/carbon-graphs/tests/unit/controls/Bubble/MultipleDataset/BubbleMultipleDatasetPanning-spec.js (hunk 4 above)
Comment (AshishMotanamGurunadham) [reply packages/carbon-graphs/tests/unit/controls/Bubble/MultipleDataset/BubbleMultipleDatasetPanning-spec.js]: Fixed [here](https://github.com/cerner/terra-graphs/pull/66/commits/1f556c88eae25ac272b14101b4dd705321f965ad#diff-d1315cea2e891de0ec24fe869f2cbeee834f4a06641aae58b0e4f1085b3e13fdR353).

File: packages/carbon-graphs/tests/unit/controls/Bubble/SingleDataset/BubbleSingleDatasetPanning-spec.js (hunk 3 above)
Comment (AshishMotanamGurunadham) [reply packages/carbon-graphs/tests/unit/controls/Bubble/SingleDataset/BubbleSingleDatasetPanning-spec.js]: Fixed [here](https://github.com/cerner/terra-graphs/pull/66/commits/1f556c88eae25ac272b14101b4dd705321f965ad#diff-10d5d54c3e7d7f730a2bdd1711f7dc6605e5d787e88735b3d019c468551026a4R354).

File: packages/carbon-graphs/CHANGELOG.md [hunk 5]
```diff
@@ -4,6 +4,7 @@
 
 * Changed
   * Fixed linter issues.
+  * Fixed Y-axis range during reflow.
```
Comment (sdadn) [inline packages/carbon-graphs/CHANGELOG.md:7]: Nit: Fixed Y-axis range *bug in* reflow.

Comment (AshishMotanamGurunadham) [reply packages/carbon-graphs/CHANGELOG.md]: Fixed [here](https://github.com/cerner/terra-graphs/pull/66/commits/c46464886605a86eea3e0af75fa9d5f26fab19e5#diff-7b73f4b73c95577a16e8a58d2e1377b2204811d315c020884e80b9f22128c42aR7).

**Review by sdadn — APPROVED** [review]

**Pull request merged**